pkg/node/manager/manager.go

```diff
@@ -833,6 +845,46 @@ func (m *manager) GetNodes() map[nodeTypes.Identity]nodeTypes.Node {
 	return nodes
 }
 
+// StartNodeNeighborLinkUpdater manages node neighbors links sync.
+// This provide a central location for all node neighbor link updates.
+// Under proper conditions, publisher enqueues the node which requires a link update.
+// This controller is agnostic of the condition under which the links must be established, thus
+// that responsibility lies on the publishers.
+// This controller also provides for module health to be reported in a single central location.
+// TODO might need to validate node is still cool?
+func (m *manager) StartNodeNeighborLinkUpdater(nh datapath.NodeNeighbors) {
+	sc := cell.GetSubScope(m.healthScope, "neighbor-link-updater")
+	controller.NewManager().UpdateController(
+		"node-neighbor-link-updater",
+		controller.ControllerParams{
+			Group: neighborTableUpdateControllerGroup,
```

Comment:
FYI: [controller.ControllerParams](https://github.com/cilium/cilium/blob/main/pkg/controller/controller.go#L69) now has a field that will automatically update health based on controller outcome.

Reply:
Thank Tom! I think this would not help here as we need an additional scope based on the node being processed.